easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2021b.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonBundle'
+
+name = 'kb-python'
+version = '0.27.3'
+
+homepage = 'https://www.kallistobus.tools/'
+description = """kallisto | bustools is a workflow for pre-processing single-cell RNA-seq data.
+ Pre-processing single-cell RNA-seq involves: (1) association of reads with their cells of origin,
+ (2) collapsing of reads according to unique molecular identifiers (UMIs), and (3) generation of
+ gene or feature counts from the reads to generate a cell x gene matrix."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('h5py', '3.6.0'),
+    ('IPython', '7.26.0'),
+    ('loompy', '3.0.7'),
+    ('Pysam', '0.17.0'),
+    ('Python', '3.9.6'),
+    ('scanpy', '1.9.1'),
```

Comment:
Version 1.8.2 of `scanpy` is already used by other easyconfigs that use the `foss/2021b` toolchain. Could you stick to that one here as well?